README.md

```diff
@@ -53,8 +53,9 @@ $ npm start
 - Build and run Docker for a service:
 ```
 $ cd ./inji-web
-$ docker build -f Dockerfile.local -t <dockerImageName>:<tag> .
-$ docker run -p 3004:3004 <dockerImageName>:<tag>
+$ docker build -f Dockerfile -t <dockerImageName>:<tag> .
```

Comment:
> docker build -f Dockerfile -t <dockerImageName>:<tag> .

Dont need to add **-f Dockerfile** use

`docker build -t <dockerImageName>:<tag> .`